ui/components/MesheryMeshInterface/PatternServiceForm.js

```diff
@@ -63,15 +64,18 @@ function createPatternFromConfig(config, namespace) {
 
   Object.keys(config).forEach((key) => {
     // Add it only if the settings are non empty or "true"
-    if (config[key].settings) pattern.services[key] = config[key];
+    if (config[key].settings) {
+      const name = PascalCaseToKebab(key);
```

Comment:
this is for the service name right?

Reply:
Yes. This should fix the Virtual Service issue